bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/i18n/TranslationProvider.java

```diff
@@ -55,6 +58,8 @@
      *
      * @return the translated text or the default text (could be null or empty)
      */
-    String getText(Bundle bundle, String key, String defaultText, Locale locale, Object... arguments);
+    @Nullable
+    String getText(@Nullable Bundle bundle, @Nullable String key, @Nullable String defaultText, @Nullable Locale locale,
```

Comment:
Didn't we agree on that we only want to state if a parameter is `NonNull` and if if can be `null` we omit the `@Nullable` annotation?